Maven build failed in code checking with latest commit 7c94229

[INFO] There is 1 error reported by Checkstyle 8.1 with jar:file:/home/admin1/.m2/repository/org/openhab/tools/sat/sat-plugin/0.5.0/sat-plugin-0.5.0.jar!/rulesets/checkstyle/rules.xml ruleset.
[INFO] 
[INFO] --- sat-plugin:0.5.0:pmd (default) @ org.openhab.core ---
[INFO] Adding dependency to pmd:0.5.0
[INFO] Adding dependency to pmd-core:5.8.1
[INFO] Adding dependency to pmd-java:5.8.1
[INFO] Adding dependency to pmd-javascript:5.8.1
[INFO] Adding dependency to pmd-jsp:5.8.1
[INFO] 
[INFO] --- sat-plugin:0.5.0:spotbugs (default) @ org.openhab.core ---
[INFO] Adding dependency to findbugs:0.5.0
[INFO] Adding dependency to bug-pattern:1.2.4
[INFO] Adding dependency to spotbugs:3.1.2
[INFO] Fork Value is false
     [java] JVM args ignored when same JVM is used.
     [java] Warnings generated: 1
[INFO] Done SpotBugs Analysis....
[INFO] 
[INFO] --- sat-plugin:0.5.0:report (default) @ org.openhab.core ---
[INFO] Individual report appended to summary report.
[ERROR] Code Analysis Tool has found: 
 1 error(s)! 
 7 warning(s) 
 10 info(s)
[WARNING] org.openhab.core.internal.CoreActivator.java:[19]
First javadoc author should have "Initial contribution" contribution description.
[WARNING] org.openhab.core.internal.CoreActivator.java:[20]
Javadoc author should not have empty contribution description.
[WARNING] org.openhab.core.internal.CoreActivator.java:[38]
Avoid using ServiceTracker for dependency injection, consider using Declarative Services
[WARNING] org.openhab.core.internal.icon.IconForwarder.java:[23]
First javadoc author should have "Initial contribution" contribution description.
[WARNING] org.openhab.core.internal.item.DefaultSitemapProvider.java:[31]
First javadoc author should have "Initial contribution" contribution description.
[WARNING] org.openhab.core.OpenHAB.java:[23]
First javadoc author should have "Initial contribution" contribution description.
[ERROR] .core/META-INF/MANIFEST.MF:[0]
**The manifest version did not match the requirements ^2\.3\.0**
[WARNING] .core/META-INF/MANIFEST.MF:[12]
The exported package `org.openhab.core` is not imported
[INFO] Detailed report can be found at: file:////home/admin1/smarthome-master/git/openhab-core/bundles/org.openhab.core/target/code-analysis/report.html
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] openHAB ............................................ SUCCESS [  0.298 s]
[INFO] openHAB - Tycho settings ........................... SUCCESS [  0.802 s]
[INFO] openHAB ............................................ SUCCESS [  0.184 s]
[INFO] openHAB Bundles .................................... SUCCESS [  0.121 s]
[INFO] openHAB Core ....................................... FAILURE [ 30.517 s]
[INFO] openHAB Core 1.x Compatibility Layer ............... SKIPPED
[INFO] openHAB Core 1.x Compatibility Layer Tests ......... SKIPPED
[INFO] openHAB Karaf Integration .......................... SKIPPED
[INFO] openHAB SSL Certificate Generator .................. SKIPPED
[INFO] openHAB Dashboard UI ............................... SKIPPED
[INFO] openHAB REST API Documentation ..................... SKIPPED
[INFO] openHAB Sound Support .............................. SKIPPED
[INFO] openHAB Basic UI Dashboard Integration ............. SKIPPED
[INFO] openHAB Classic UI Dashboard Integration ........... SKIPPED
[INFO] openHAB HomeBuilder UI Dashboard Integration ....... SKIPPED
[INFO] openHAB Paper UI Theme Fragment .................... SKIPPED
[INFO] openHAB Features ................................... SKIPPED
[INFO] openHAB Karaf Features ............................. SKIPPED
[INFO] openHAB Karaf Feature Verification ................. SKIPPED
[INFO] openHAB p2 Feature ................................. SKIPPED
[INFO] openHAB Core P2 Repository ......................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------

Any idea ?

--- a/tools/checkstyle.properties
+++ b/tools/checkstyle.properties
@@ -1,2 +1,3 @@
 checkstyle.headerCheck.content=^/\\*\\*$\\n^ \\* Copyright \\(c\\) {0}-{1} by the respective copyright holders\\.$\\n^ \\*$\\n^ \\* All rights reserved\\. This program and the accompanying materials$\\n^ \\* are made available under the terms of the Eclipse Public License v1\\.0$\\n^ \\* which accompanies this distribution, and is available at$\\n^ \\* http://www.eclipse.org/legal/epl\\-v10\\.html$
 checkstyle.headerCheck.values=2015,2018 
+checkstyle.pomXmlCheck.currentVersionRegex=^2\.4\.0

